todo/src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import './App.css';
+import AddTodo from './components/AddTodo';
+import TodoList from './components/TodoList';
+
+export type todoType = {
+        task: string;
+        completed : boolean
```

Comment:
if it boolean value then add prefix as is like isCompleted

Reply:
changes done